webui/react/src/hooks/useModal/UserSettings/useModalCreateUser.tsx

```diff
@@ -41,30 +45,13 @@ interface FormValues {
   USER_NAME_NAME: string;
 }
 
-const permissionColumns = [
-  {
-    dataIndex: 'name',
-    key: 'name',
-    title: 'Name',
-  },
-  {
-    dataIndex: 'isGlobal',
-    key: 'isGlobal',
-    render: (val: boolean) => (val ? <Icon name="checkmark" /> : ''),
-    title: 'Global?',
-  },
-  {
-    dataIndex: 'workspaceOnly',
-    key: 'workspaceOnly',
-    render: (val: boolean) => (val ? '' : <Icon name="checkmark" />),
-    title: 'Workspaces?',
-  },
-];
-
 const ModalForm: React.FC<Props> = ({ form, branding, user, groups, viewOnly }) => {
-  const [permissions, setPermissions] = useState<Permission[]>([]);
+  const [ permissions, setPermissions ] = useState<Permission[]>([]);
 
+  const rbacEnabled = useFeature().isOn('rbac');
   const canSeePermissions = usePermissions().canGetPermissions;
```

Comment:
You should call `usePermissions` just once. Then you can do `if (user && permissions.canSeePermissions)` etc.

Reply:
Makes sense, thank you